package.json

```diff
@@ -142,6 +144,10 @@
       "pre-commit": "npm run lint"
     }
   },
+  "prettier": {
```

Comment:
Can you use these settings?

```json
{
  "arrowParens": "always",
  "bracketSpacing": false,
  "quoteProps": "preserve",
  "singleQuote": true
}
```

Reply:
Do you perhaps want `"quoteProps": "consistent",`? That seems to fix the two issues highlighted below.

Reply:
No, looking at [an internal discussion](https://groups.google.com/a/google.com/g/javascript-style/c/Ob2Erbr29Uw/m/M2Unems8BgAJ) on prettier preferences, "preserve" is recommended because of how Closure Compiler does property minification. It's not safe to add or remove quote on object properties as that would potentially break the code after its compiled, and since this library is used in google3, I think it's better to not change this.